templates/report.js

```diff
@@ -55,6 +55,16 @@ angular.module('app', [])
 
       traverse(results);
 
+      var startSec = results.startTime[0] + (results.startTime[1] * (1e-9)),
+      endSec = results.endTime[0] + (results.endTime[1] * (1e-9)),
+      secDiff = Math.round(endSec - startSec);
+
+      metrics.timeMin = Math.floor(secDiff / 60);
+      metrics.timeSec = Math.floor(secDiff - (metrics.timeMin * 60));
+
+      metrics.timeMin = metrics.timeMin < 10 ? "0" + metrics.timeMin : metrics.timeMin;
```

Comment:
this is nice, 

another option:
if (metrics.timeMin < 10) { metrics.timeMin = '0' + metrics.timeMin }